### PR TITLE
Basic particles

### PR DESCRIPTION
**Pull request by Cutaiar**
Adapted our THREE particle system from spotiverse to React and integrated it with Trentons data access change.

Note: did not hook up songs to particles yet (they are random). That change coming in next PR.

### CODE REVIEW

File: frontend/src/models/artist.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export type Artist = {
+  followers: number;
+  genres: string[];
+  mainArtist: string;
+  featuredArtists: string[];
+  popularity: number;
+  images: string; // TODO why did this used to be Array<Map<string, any>>
```
Comment (mtrent6) [inline frontend/src/models/artist.ts:7]: ya this is a bug on my end that i thought i fixed, it should be an image[] where image is an object like:

`image = {
url: string;
height: number;
width: number;
}`

File: frontend/src/components/ThreeEngine/Particle.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import * as THREE from "three";
+
+export class Particle {
```
Comment (afitz4) [inline frontend/src/components/ThreeEngine/Particle.js:3]: Can we eventually convert this file to TS to make it strongly typed or is that a nono?

**Review by afitz4 — COMMENTED** [review]
lfg

**Comment by mtrent6** [comment]
will fix warnings later, should probably turn off netlify feature to not treat warnings as errors for now while developing

File: frontend/src/components/ThreeEngine/Particle.js (hunk 2 above)
Comment (Cutaiar) [reply frontend/src/components/ThreeEngine/Particle.js:3]: yeah def should be ts

**Pull request merged**